### PR TITLE
Skill improvments

### PR DESCRIPTION
**Pull request by india-kerle**
Ok, I spent ages trying to do something fancy then got v scared so now i'm just hard coding skills at the very end - i've added the hard code file to both the public + private buckets.

### CODE REVIEW

File: ojd_daps_skills/pipeline/extract_skills/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 `extract_skills.py` combines the prediction of skills using code from [skill_ner](https://github.com/nestauk/ojd_daps_skills/tree/dev/ojd_daps_skills/pipeline/skill_ner) with the mapping of skills to a taxonomy using code from [skill_ner_mapping](https://github.com/nestauk/ojd_daps_skills/tree/dev/ojd_daps_skills/pipeline/skill_ner_mapping).
 
+If you are mapping to the ESCO skills taxonomy, we have hard coded the top 100 skills from a random sample of 100,000 job adverts in OJO with the most appropriate skills from the taxonomy.
```
Comment (lizgzil) [inline ojd_daps_skills/pipeline/extract_skills/README.md]: Kind of sounds worse than it is. We only hardcoded 43 of the most common skills which we thought weren't well matched

File: ojd_daps_skills/pipeline/skill_ner_mapping/hard_coded_mapper_formatting.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+if __name__ == "__main__":
+
+    s3 = get_s3_resource()
+
+    hard_coded_skills = (
+        load_s3_data(
+            s3,
+            bucket_name,
+            "escoe_extension/inputs/data/skill_mappings/hard_labelled_skills.csv",
```
Comment (lizgzil) [inline ojd_daps_skills/pipeline/skill_ner_mapping/hard_coded_mapper_formatting.py:22]: I noticed this csv has some unwanted tabs in it which was causing `hardcoded_ojo_esco_lookup.json` to have things like `{"match_skill": "picking orders for dispatching\t", "match_id": "ee5e2297-39cc-4ed6-89e5-d58016118a7f\t"}`. 
So I've replaced "hard_labelled_skills.csv" with a cleaned up version

Comment (lizgzil) [reply ojd_daps_skills/pipeline/skill_ner_mapping/hard_coded_mapper_formatting.py:22]: it only occured for "ojo_skill": "Pick our customer orders" and "ojo_skill": "pack our customer orders"

Comment (lizgzil) [reply ojd_daps_skills/pipeline/skill_ner_mapping/hard_coded_mapper_formatting.py:22]: I kept your original file in S3 but renamed it s3://open-jobs-lake/escoe_extension/inputs/data/skill_mappings/hard_labelled_skills_unclean.csv (please delete if you don't have a need for it)

**Review by lizgzil — APPROVED** [review]
this looks good and I don't think it is introducing any bugs!
```
pytest
```
works for me. 
I've:
- Added a new test for hardcode, this way it just checks the mapping has been done to whats specified in the hardcode file, rather than the extracting skills from a made up advert.
- Checked pytest works
- Fixed a bug I found
- Updated `hardcoded_ojo_esco_lookup.json` in public and private buckets (with the instances of '\t' taken out)

File: ojd_daps_skills/pipeline/extract_skills/README.md (hunk 1 above)
Comment (india-kerle) [reply ojd_daps_skills/pipeline/extract_skills/README.md]: ya true

**Comment by india-kerle** [comment]
ok fabbo @lizgzil - I've modified the readme to say we only hard coded 43 skills - otherwise sounds like its good to merge

**Pull request merged**